.drone.yml

```diff
@@ -0,0 +1,111 @@
+---
+kind: pipeline
+name: preview
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+  - name: install
+    image: node:18-alpine
+    commands:
+      - yarn install --frozen-lockfile
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
+
+  - name: ignore-openapi
+    image: node:18-alpine
+    commands:
+      - apk add git
+      - ./scripts/ignore-openapi
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
+
+  - name: build
+    image: node:18-alpine
+    commands:
+      - yarn build
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
+
+  - name: package
+    image: registry.suse.com/bci/bci-base:15.4
+    commands:
+      - zypper install -y zip
+      - zip -q -r build.zip build/
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
+
+  - name: preview
+    image: ghcr.io/harvester/drone-netlify:master
+    environment:
+      BUILD_ZIP_FILE: "build.zip"
+      NETLIFY_AUTH_TOKEN:
+        from_secret: NETLIFY_AUTH_TOKEN
+      NETLIFY_SITE_ID:
+        from_secret: NETLIFY_SITE_ID
+      GITHUB_TOKEN:
+        from_secret: GITHUB_TOKEN
+
+volumes:
+  - name: docker
+    host:
+      path: /var/run/docker.sock
+
+trigger:
+  event:
+    - pull_request
+
+---
+kind: pipeline
+name: release
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+  - name: install
+    image: node:18-alpine
+    commands:
+      - yarn install --frozen-lockfile
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
+
+  - name: build
+    image: node:18-alpine
+    commands:
+      - yarn build
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
+
+  - name: release
+    image: node:18-alpine
+    environment:
+      GITHUB_TOKEN:
+        from_secret: GITHUB_TOKEN
+    commands:
+      - apk add git
+      - npm install -g gh-pages
+      - ./scripts/gh-pages
```

Comment:
`plugins/gh-pages` connect to GitHub API by username and password, not GitHub Token.

Reply:
but we are not using the GH action, only its container image and gh-pages cli; if the base image works, this will save our build time from the package installation.

Reply:
Not find an official base image for the `gh-pages` command. Most of it is for GitHubAction, `plugins/gh-pages` is different from the `gh-pages` command.